client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -207,6 +207,125 @@ export const useCommandsArrayWpcom = ( {
+				},
+			},
+		},
+		{
+			name: 'registerDomain',
+			label: __( 'Register domain' ),
+			searchLabel: __( 'register domain' ),
+			context: 'Registering domain',
+			callback: ( { close }: { close: () => void } ) => {
+				close();
+				navigate( `/start/domain/domain-only` );
+			},
+			icon: domainsIcon,
+		},
+		{
+			name: 'openActivityLog',
+			label: __( 'Open Activity Log' ),
+			searchLabel: __( 'open activity log' ),
+			context: 'Opening activity log',
+			callback: setStateCallback( 'openActivityLog' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/activity-log/${ site.slug }` );
+				},
+			},
+		},
+		{
+			name: 'openBackups',
+			label: __( 'Open backups' ),
+			searchLabel: __( 'open backups' ),
+			context: 'Opening backups',
+			callback: setStateCallback( 'openBackups' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/backup/${ site.slug }` );
+				},
+			},
+		},
+		{
+			name: 'viewSiteMetrics',
+			label: __( 'View site metrics' ),
+			searchLabel: __( 'view site metrics' ),
+			context: 'Viewing site metrics',
+			callback: setStateCallback( 'viewSiteMetrics' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/site-monitoring/${ site.slug }` );
+				},
+			},
+			filter: ( site: SiteExcerptData ) => site?.is_wpcom_atomic,
+		},
+		{
+			name: 'openPHPLogs',
+			label: __( 'Open PHP logs' ),
+			searchLabel: __( 'open PHP logs' ),
+			context: 'Opening PHP logs',
+			callback: setStateCallback( 'openPHPLogs' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/site-monitoring/${ site.slug }/php` );
+				},
+			},
+			filter: ( site: SiteExcerptData ) => site?.is_wpcom_atomic,
+		},
+		{
+			name: 'openWebServerLogs',
+			label: __( 'Open web server logs' ),
+			searchLabel: __( 'open web server logs' ),
+			context: 'Opening web server logs',
+			callback: setStateCallback( 'openWebServerLogs' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/site-monitoring/${ site.slug }/web` );
+				},
+			},
+			filter: ( site: SiteExcerptData ) => site?.is_wpcom_atomic,
+		},
+		{
+			name: 'openHostingConfiguration',
+			label: __( 'Open hosting configuration' ),
+			searchLabel: __( 'open hosting configuration' ),
+			context: 'Opening hosting configuration',
+			callback: setStateCallback( 'openHostingConfiguration' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/hosting-config/${ site.slug }` );
+				},
+			},
+			filter: ( site: SiteExcerptData ) => ! isNotAtomicJetpack( site ),
```

Comment:
@sejas for this command, I think we need a check to filter out the sites that are just Jetpack sites since Hosting Configuration is not available there. What do you think?

Reply:
True!, Let's also remove the P2s.

https://github.com/Automattic/wp-calypso/blob/5737064c77c0dd2240af51c0bf51ddcce57eb1f8/client/sites-dashboard/components/sites-ellipsis-menu.tsx#L404-L404

Reply:
Makes sense! I was looking at the same code for reference 👍